pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -711,6 +711,11 @@ func (r *ReconcileCredentialsRequest) Reconcile(ctx context.Context, request rec
 
 				logger.Errorf("errored with condition: %v", t.Reason())
 				r.updateActuatorConditions(cr, t.Reason(), syncErr)
+				// Update the status of the CredentialsRequest object
+				err := r.Client.Status().Update(ctx, cr)
```

Comment:
@fxierh This is where you meant to call Update, right?

Reply:
Yep